### PR TITLE
Fix timed external conditions being rejected instead of reset.

### LINKED ISSUE #12498: Timed external conditions cannot be reset
e.g. the Iron Curtain in RA.  It should be possible to apply a second curtain charge to an already curtained unit to reset the timer.

### PR DESCRIPTION
**Pull request by pchote**
Fixes #12498.

### CODE REVIEW

**Review by GraionDilach — APPROVED** [review]
:+1:

File: OpenRA.Mods.Common/Scripting/Properties/ConditionProperties.cs [hunk 1]
```diff
@@ -83,7 +83,7 @@ public void GrantTimedUpgrade(string upgrade, int duration)
 		public bool AcceptsUpgrade(string upgrade)
 		{
 			Game.Debug("AcceptsUpgrade is deprecated. Use AcceptsCondition instead.");
-			return AcceptsCondition(upgrade);
+			return AcceptsCondition(upgrade, false);
```
Comment (Phrohdoh) [inline OpenRA.Mods.Common/Scripting/Properties/ConditionProperties.cs]: Why is this explicit `false` necessary?

Comment (pchote) [inline OpenRA.Mods.Common/Scripting/Properties/ConditionProperties.cs]: It isn't on this version of the commit.  Removed.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/b85a6ffd7548a1305a81d0788014d4e82e7418cd...8d908672239a08e89a7acdbc09fc0f39e53fd675)

**Pull request merged**